src/Components/ImagePost.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "../CSS/ImagePost.css";
+
+const ImagePost = () => {
+  return (
+    <>
+      <div className="photo">
+        <div className="photoDetails">
+          <div className="photoImage">
+            <img
+              src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9e/Plus_symbol.svg/500px-Plus_symbol.svg.png"
+              alt=""
+              id="photoImage"
+            />
+            Add Image Link
+          </div>
+          <div className="photoRight">
+            <div className="imageTitle">Post Title :</div>
+            <input id="imageTitle" placeholder="Image Title" />
+            <div className="imageToolsUsed">Tools Used :</div>
+            <input id="imageToolsUsed" placeholder="Tools Used" />
+            <div className="imageArtist">Artist Name :</div>
+            <input id="imageArtist" placeholder="Artist Name" />
+          </div>
+        </div>
+        <div id="filters">Filters :</div>
+        <div className="filters">
+          <label class="container">
+            <input type="checkbox" id="checkbox"/>
+            Simple Design
+          </label>
+          <label class="container">
```

Comment:
You are replicating the same components, try to make an separate component and reuse it using map function.